<launch>

    <rosparam file="low_level_control.yaml" command="load"/>

    <node name="robot_arm_controller_spawner" pkg="controller_manager" type="spawner"
          respawn="true" output="screen"
          args="/robopi/joint_state_controller
                /robopi/diff_drive_controller"/>


</launch>




